src/main/java/laddergame/domain/LinkPicker.java

```diff
@@ -0,0 +1,23 @@
+package laddergame.domain;
+
+import java.security.NoSuchAlgorithmException;
+import java.security.SecureRandom;
+import java.util.Random;
+
+public class LinkPicker implements PickStrategy {
```

Comment:
유독 네이밍에 대한 리뷰를 많이 달게 되네요 ㅎㅎ 네이밍은 개인의 취향에 의해 많이 갈리는 부분이라, 참고만 해주세요!! :) 
```suggestion
public class RandomLinkGenerator implements LinkGenerator {
```
발판(링크)을 생성하는 전략에 대한 객체이니, 위와 같은 이름은 어떨까요?
어떤 역할을 하는 인터페이스인지, 또 어떤 방식의 구현체인지 이해하기 좋을 것 같네요.